src/pages/iou/propTypes/index.js

```diff
@@ -21,6 +21,9 @@ const iouPropTypes = PropTypes.shape({
     /** The category name */
     category: PropTypes.string,
 
+    /** The Billable */
```

Comment:
```suggestion
    /** Whether the request is billable */
```

Sorry for the nitpicks here, just trying to make it as descriptive as possible for those who don't really know what it is